config-model/src/main/java/com/yahoo/vespa/model/container/http/xml/HttpBuilder.java

```diff
@@ -121,7 +121,8 @@ private static AthenzDomain getAccessControlDomain(DeployState deployState, Elem
 
                 for (Element bindingSpec: XML.getChildren(child, "binding")) {
                     String binding = XML.getValue(bindingSpec);
-                    result.add(FilterBinding.create(chainId, UserBindingPattern.fromPattern(binding)));
+                    FilterBinding.Type type = tagName.equals("request-chain") ? FilterBinding.Type.REQUEST : FilterBinding.Type.RESPONSE;
```

Comment:
Would prefer explicit check for `"response-chain"` and throw otherwise, just to be absolutely future-proff.

Reply:
I have improved the logic in new commit.